spec/rails_helper.rb

```diff
@@ -0,0 +1,72 @@
+# simplecov on demand coverage spec.
+# run with "COVERAGE=true bundle exec rake spec"
+if ENV['COVERAGE']
+  require 'simplecov'
+  require 'coveralls'
+
+  SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new(
+    [SimpleCov::Formatter::HTMLFormatter, Coveralls::SimpleCov::Formatter]
+  )
+  SimpleCov.start 'rails' do
+    add_filter '/spec'
+    add_group 'Models', 'app/models'
+    add_group 'Controllers', 'app/controllers'
+    coverage_dir File.join('coverage', Time.now.strftime('%Y%m%d-%H%M%S'))
+  end
+end
+
```

Comment:
Would you mind to add a comment that the Coverage things need to be at the top of the file? 
I had to look that up, because it looks contrary to line 26 "Add additional requires below this line".

Reply:
🙇  thanks :-)